src/TrackPanel.cpp

```diff
@@ -350,18 +350,13 @@ AudacityProject * TrackPanel::GetProject() const
    //Do it in two stages in case 'this' is not a wxWindow.
    //when the compiler will flag the error.
    wxWindow const * const pConstWind = this;
-   wxWindow * pWind=(wxWindow*)pConstWind;
-#ifdef EXPERIMENTAL_NOTEBOOK
```

Comment:
Let's get rid of the rest of EXPERIMENTAL_NOTEBOOK in the source tree.  That includes the line in Experimental.cmake.  The rest of the source code isn't available to turn it on again even if we wanted to.

Reply:
Will do